gmakefile

```diff
@@ -12,7 +12,7 @@ libtdycore_static := $(LIBDIR)/libtdycore.$(AR_LIB_SUFFIX)
 libtdycore := $(if $(filter-out no,$(BUILDSHAREDLIB)),$(libtdycore_shared),$(libtdycore_static))
 
 pkgs := tdycore
-srcs-tdycore.c   := $(wildcard src/mpfao/*.c src/fe/*.c src/materials/*.c src/*.c src/f90-mod/*.c)
```

Comment:
So does this mean we're just not going to compile the finite element code for now?

Reply:
We are still building the FE code. I added a new dir `src/fvtpf/*.c`

Reply:
Ah, right. I see now.